ambry-replication/src/test/java/com/github/ambry/replication/ReplicationTest.java

```diff
@@ -1328,6 +1335,94 @@ public void replicaThreadTestConverter() throws Exception {
         idsToBeIgnoredByPartition, storeKeyConverter, expectedIndex, expectedIndex, 5);
   }
 
+  /**
+   * Tests if deprecated containers have been blocked during replication.
+   */
+  @Test
+  public void blockDeprecatedContainerReplicationTest() throws Exception {
```

Comment:
The ReplicationSkipPredicate class has low code coverage because the test only runs the positive case where container is skipped.  Please add negative test cases.

Reply:
See https://codecov.io/gh/linkedin/ambry/pull/1547/diff?src=pr&el=tree#diff-YW1icnktYXBpL3NyYy9tYWluL2phdmEvY29tL2dpdGh1Yi9hbWJyeS9yZXBsaWNhdGlvbi9SZXBsaWNhdGlvblNraXBQcmVkaWNhdGUuamF2YQ==